logstash-core/src/main/java/org/logstash/Logstash.java

```diff
@@ -38,6 +38,8 @@ public static void main(final String... args) {
                     "LS_HOME environment variable must be set. This is likely a bug that should be reported."
             );
         }
+        removeNashornDeprecationSwitchForJavaOltherThan11();
```

Comment:
can we do the opposite, and add the property if java is above 8?

Reply:
The property was introduced in JDK 11, so we should check also Java 10 and 9, I think

Reply:
Done :-) inverted the logic